src/vault_snapshot/vault_snapshot/snap.py

```diff
@@ -3,49 +3,32 @@
 
```

Comment:
I don't understand why this file is in a submodule with the same name as the parent... that is not convention. More common to find `utils` sub-modules, for example. I would just move this function implementation to the parent `vault_snapshot/snap.py`... there's no need for this sub-folder.

Reply:
yeah, `vault_snapshot` was written while back in this old convention. i'll bring it up to new standard.

Reply:
[updated](https://github.com/artsy/opstools/pull/142/commits/ff93ca2d22544b386f906c4c5a418748c7605a35).